File: Android/sources/androidx/media3/exoplayer/dash/DashMediaSource$Factory.java
package androidx.media3.exoplayer.dash;

import androidx.work.WorkRequest;
import java.util.List;

public final class DashMediaSource$Factory implements lz8 {
    public final cs a;
    public final g74 b;
    public final jk3 c = new jk3(6);
    public final p9a d = new p9a(10);
    public final nfd e = new nfd(10);
    public final long f = WorkRequest.DEFAULT_BACKOFF_DELAY_MILLIS;
    public final long g = 5000000;

    public DashMediaSource$Factory(g74 g74) {
        cs csVar = new cs(g74);
        this.a = csVar;
        this.b = g74;
        ((qf4) csVar.o).c = true;
    }

    public final lz8 a() {
        n79.k((Object) null, "MediaSource.Factory#setDrmSessionManagerProvider no longer handles null by instantiating a new DefaultDrmSessionManagerProvider. Explicitly construct and pass an instance in order to retain the old behavior.");
        throw null;
    }

    public final void b(boolean z) {
        ((qf4) this.a.o).c = z;
    }

    public final lz8 c() {
        n79.k((Object) null, "MediaSource.Factory#setLoadErrorHandlingPolicy no longer handles null by instantiating a new DefaultLoadErrorHandlingPolicy. Explicitly construct and pass an instance in order to retain the old behavior.");
        throw null;
    }

    public final void e(ece ece) {
        ece.getClass();
        qf4 qf4 = (qf4) this.a.o;
        qf4.getClass();
        qf4.b = ece;
    }

    /* renamed from: f */
    public final o64 d(ir8 ir8) {
        ir8.b.getClass();
        o8b c64 = new c64();
        List list = ir8.b.e;
        return new o64(ir8, this.b, !list.isEmpty() ? new npg(12, (Object) c64, (Object) list) : c64, this.a, this.d, this.c.j(ir8), this.e, this.f, this.g);
    }
}

File: IOS/v2/iosx/media3/exoplayer/dash/DashMediaSource$Factory.swift
package iOSx.media3.exoplayer.dash;.swift
import iOSx.work.WorkRequest;.swift
import java.util.List;.swift
public final class DashMediaSource$Factory implements lz8 {.swift
public final cs a;.swift
public final g74 b;.swift
public final jk3 c = new jk3(6);.swift
public final p9a d = new p9a(10);.swift
public final nfd e = new nfd(10);.swift
public final long f = WorkRequest.DEFAULT_BACKOFF_DELAY_MILLIS;.swift
public final long g = 5000000;.swift
public DashMediaSource$Factory(g74 g74) {.swift
cs csVar = new cs(g74);.swift
this.a = csVar;.swift
this.b = g74;.swift
((qf4) csVar.o).c = true;.swift
}.swift
public final lz8 a() {.swift
n79.k((Object) null, "MediaSource.Factory#setDrmSessionManagerProvider no longer handles null by instantiating a new DefaultDrmSessionManagerProvider. Explicitly construct and pass an instance in order to retain the old behavior.");.swift
throw null;.swift
}.swift
public final void b(boolean z) {.swift
((qf4) this.a.o).c = z;.swift
}.swift
public final lz8 c() {.swift
n79.k((Object) null, "MediaSource.Factory#setLoadErrorHandlingPolicy no longer handles null by instantiating a new DefaultLoadErrorHandlingPolicy. Explicitly construct and pass an instance in order to retain the old behavior.");.swift
throw null;.swift
}.swift
public final void e(ece ece) {.swift
ece.getClass();.swift
qf4 qf4 = (qf4) this.a.o;.swift
qf4.getClass();.swift
qf4.b = ece;.swift
}.swift
/* renamed from: f */.swift
public final o64 d(ir8 ir8) {.swift
ir8.b.getClass();.swift
o8b c64 = new c64();.swift
List list = ir8.b.e;.swift
return new o64(ir8, this.b,  c64, this.a, this.d, this.c.j(ir8), this.e, this.f, this.g);.swift
}.swift
}.swift
